editor/filesystem_dock.cpp

```diff
@@ -3382,17 +3406,23 @@ FileSystemDock::FileSystemDock() {
 
 	add_child(tree_popup);
 
-	split_box = memnew(VSplitContainer);
-	split_box->set_v_size_flags(SIZE_EXPAND_FILL);
-	add_child(split_box);
+	vsplit_box = memnew(VSplitContainer);
+	vsplit_box->set_v_size_flags(SIZE_EXPAND_FILL);
+	add_child(vsplit_box);
+	split_box = vsplit_box;
+
+	hsplit_box = memnew(HSplitContainer);
```

Comment:
You could just use `SplitContainer` and toggle `vertical` I think.

Reply:
@KoBeWi Thanks for the review
Fixed